lua/nvim-tree/explorer/reload.lua

```diff
@@ -18,6 +18,15 @@ local function update_status(nodes_by_path, node_ignored, status)
   end
 end
 
+local function table_find(table, value)
```

Comment:
This is useful. Please move to utils:

```lua
function M.array_index_of(array, item)
```

Reply:
Actually `utils.array_remove` is exactly what I need